source/installation/deploy-manage-minio-distributed.rst

```diff
@@ -0,0 +1,620 @@
+            :class: copyable
+  
+            cat /dev/urandom | head -c 32 | base64 -
+
+         Copy the key to a secure location. MinIO cannot decode the backend
+         without this key.
+
+   * - ``minio{1...4}.example.com/``
+     - The DNS hostname of each server in the deployment.
+
+       The command uses MinIO expansion notation ``{x...y}`` to denote a
+       sequential series. Specifically, the hostname
+       ``https://minio{1...4}.example.com`` expands to:
+  
+       - ``https://minio1.example.com``
+       - ``https://minio2.example.com``
+       - ``https://minio3.example.com``
+       - ``https://minio4.example.com``
+
+   * - ``minio{5...8}.example.com/``
+     - The DNS hostname of each server in the new server pool.
+
+       The command uses MinIO expansion notation ``{x...y}`` to denote a
+       sequential series. Specifically, the hostname
+       ``https://minio{1...4}.example.com`` expands to:
+  
+       - ``https://minio1.example.com``
+       - ``https://minio2.example.com``
+       - ``https://minio3.example.com``
+       - ``https://minio4.example.com``
+
+   * - ``/mnt/disk{1...4}/data``
+     - The path to each disk on the host machine. 
+
+       ``/data`` is an optional folder in which the ``minio`` server stores
+       all information related to the deployment. 
+
+       The command uses MinIO expansion notation ``{x...y}`` to denote a
+       sequential series. Specifically,  ``/mnt/disk{1...4}/data`` expands to:
+      
+       - ``/mnt/disk1/data``
+       - ``/mnt/disk2/data``
+       - ``/mnt/disk3/data``
+       - ``/mnt/disk4/data``
+
+       See :mc-cmd:`minio server DIRECTORIES` for more information on
+       configuring the backing storage for the :mc:`minio server` process.
+
+   * - ``--console-address ":9001"``
+     - The static port on which the embedded MinIO Console listens for incoming
+       connections.
+
+       Omit to allow MinIO to select a dynamic port for the MinIO Console. 
+       Browsers opening the root node hostname 
+       ``https://minio1.example.com:9000`` are automatically redirected to the
+       Console.
+
+You may specify other :ref:`environment variables 
+<minio-server-environment-variables>` as required by your deployment.
+
+4) Open the MinIO Console
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open your browser and access any of the MinIO hostnames at port ``:9001`` to
+open the MinIO Console login page. For example,
+``https://minio1.example.com:9001``.
+
+Log in with the :guilabel:`MINIO_ROOT_USER` and :guilabel:`MINIO_ROOT_PASSWORD`
+from the previous step.
+
+.. image:: /images/minio-console-dashboard.png
+   :width: 600px
+   :alt: MinIO Console Dashboard displaying Monitoring Data
+   :align: center
+
+You can use the MinIO Console for general administration tasks like
+Identity and Access Management, Metrics and Log Monitoring, or 
+Server Configuration. Each MinIO server includes its own embedded MinIO
+Console.
+
+Deployment Recommendations
```

Comment:
@eco-minio this is mostly taken from the other published docs sites. LMK if you think anything needs tweaking.